core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/GetWalksHandler.java

```diff
@@ -45,23 +48,27 @@
 import java.util.stream.Collectors;
 
 /**
- * <p>
  * An operation handler for {@link GetWalks} operations.
- * </p>
- * <p>
- * Currently the handler only supports creating {@link Walk}s which contain
- * {@link Edge}s.
- * </p>
+ *
+ * Currently the handler only supports creating {@link Walk}s which contain {@link Edge}s.
+ *
+ * This operation handler can be modified by supplying an operationDeclarations.json
+ * file in order to limit the maximum number of hops permitted.
  */
 public class GetWalksHandler implements OutputOperationHandler<GetWalks, Iterable<Walk>> {
```

Comment:
Please can you add a some information to explain when pruning should be enabled and the effects it will have.

Reply:
Added more description to the class level Javadoc.